front_end/content.py

```diff
@@ -2006,10 +2009,14 @@ def rebuild_exercises(self, assignment_title):
 
             text_output, image_output, tests = exec_code(self.__settings_dict, exercise_details["answer_code"], exercise_basics, exercise_details)
 
+            tests_dict = []
+            for i in range(len(tests)):
+                tests_dict.append({**tests[i], **exercise_details["tests"][i]})
```

Comment:
For consistency with code we're using elsewhere, can we change this to use syntax more like the following:

```
for i in range(len(tests)):
    tests_dict[tests[i]] = exercise_details["tests][i]
```

Reply:
I'm unfamiliar with this syntax, you'll have to show me how this works in our meeting.